workshops/custom-search/README.md

```diff
@@ -0,0 +1,102 @@
+---
+name: Rajan Agarwal
+description: A custom search engine 
+author: '@rajan'
+img: 'https://cloud-9273tibqr-hack-club-bot.vercel.app/0demo.png'
+---
+
+# Custom Search
+
+Let's use Google to build Google. In this workshop, we're going to use Google's Search API to fetch queries and output the most relevant topics! Here's what the final product will look like (I highly encourage you to customize it to make it your own!)
+
+![Demo](img/demo.png)
+
+To build this product, we're going to need:
+- HTML
+- Javascript
+- CSS
+
+We'll also be delving into Google APIs, so make sure that you have an account set up! We're going to build this product on [repl.it](https://replit.com) because it sets up automatic deployments right from your web browser. 
+
+## Get Started
+
+Head on over to the [Replit Starter](https://replit.com/@rajnagrwl/search-starter) for this project and click **fork** in the top right corner. This will give you access to the styles and preliminary code to get you started!
+
+![Starter](img/preview.png)
+
+Once this is ready, head on over to the `index.html` file and add the following inbetween the `<body></body>` tags:
+
+```html
+<img src="moogle.png" width="525" class="image">
+    <div class="search">
+      <input type="text" id="search" class="input">
+      <button id="submit" class="searchbtn" onclick="submit()">Search</button>
+    </div>
+		<div style="height: 300px;"></div>
+    <div id="buttons" class="buttons">
+      <button id="allbutton" class="all" onclick="submit()"></button>
+    </div>
+    <div id="content"></div>
+    <script src="script.js"></script>
+    <script id="query" src="query.js"></script>
+```
+
+Let's break this down. First, we are adding the image via the `img` tag at the very top of the page. Then, we are adding an `input` and `button` to add a query for our search engine to interpret. Adding `onclick="submit()` let's our Javascript know to run that function when clicked! Next, we have a filter for all of our searches to appear at once. This content will then appear in the `<div id="content"></div>` tag. Finally, we are referencing our `script.js` and `query.js` files. Once completed, click 'Run' at the top of your screen and you should see the following screen.
```

Comment:
Instead of doing a large code block and then a paragraph of explanation could you break up to be a bit like:

```
small code block (1-2 lines)
```
explanation
```
small code block (1-2 lines)
```
explanation

Also the indentation on the code is a bit jank rn.